### DISCUSSION TITLE
Rendering and Selection do not agree on definition for `tab` (⇥)

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#53041] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

1. Visit https://github.com/zabbix/zabbix/blob/46c53da92afe45b22147e0528b21b47cf995d69b/src/libs/zbxdb/db.c#L122
2. Select:
    ```c

	static void	zbx_db_errlog(zbx_err_codes_t zbx_errno, int db_errno, const char *db_error, const char *context)
	{
		char	*s;

		last_db_errcode = zbx_errno;

		if (NULL != db_error)
    ```    
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/2119212/232575864-21568225-10fb-4592-80de-00072607faad.png">

The code renderer has defined a ⇥ as 8 characters (→), whereas the selection renderer has defined a ⇥ as jumping to the next tab stop (⇥) where each tab stop is +8 characters from the previous tab stop.

Either behavior on its own would be fine, both behaviors for content that is supposed to be rendered together is rather unfortunate.

**Comment by colinwm** [comment, marked as answer]
Thanks for the report @jsoref, sorry for the bug. We're aware of it and are working on fixing it.
